pages/gallery.html

```diff
@@ -1,12 +1,52 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <base href="../">
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Gallery</title>
+  <link rel="stylesheet" href="styles/style.css">
+  <link rel="stylesheet" href="styles/gallery.css">
 </head>
 <body>
-  
+  <header>
+    <h1>Otter Gallery</h1>
+    <nav>
+      <ul>
+        <li><a href="index.html">Home Page</a></li>
+        <li><a href="pages/facts.html">Otter Fun Facts</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section>
+      <figure>
+        <img src="images/GroomingSeaOtter.jpg" width="400" height="400" alt="Grooming Sea Otter"><figcaption>Grooming Sea Otter</figcaption>
+      </figure>
```

Comment:
Prefer that the formatting looks like this instead: 
```HTML
      <figure>
        <img src="images/GroomingSeaOtter.jpg" width="400" height="400" alt="Grooming Sea Otter">
        <figcaption>Grooming Sea Otter</figcaption>
      </figure>
```

Putting each sibling element on it's own new line is easier to read.